modules/gittar/helper/git_cmd.go

```diff
@@ -75,6 +76,17 @@ func runCommand2(w io.Writer, cmd *exec.Cmd, readers ...io.Reader) {
 		logrus.Printf("[ERROR] get command stdout error %v", err)
 		return
 	}
+	stderr, err := cmd.StderrPipe()
+	if err != nil {
+		logrus.Printf("[ERROR] get command stderr error %v", err)
+		return
+	}
+	scanner := bufio.NewScanner(stderr)
+	go func() {
+		for scanner.Scan() {
+			logrus.Printf("[ERROR] stderr > %s\n", scanner.Text())
```

Comment:
if there are multiple cmd with stderr at the same time, logs will be confused and strung together.

you can add a uuid for each cmd, or print all stderr one line.

Reply:
done